xbmc/addons/kodi-addon-dev-kit/doxygen/Modules/modules_python.dox

```diff
@@ -158,6 +158,51 @@ web applications or frameworks for the Python programming language.
       "",
       <b>xbmc.monitor().onDatabaseScanStarted()</b> function was removed completely.
 }
+\python_removed_function{
+      onDatabaseScanStarted,
+      "",
+      <b>xbmc.monitor().onAbortRequested()</b> function was removed completely.
+}
+\python_removed_class{
```

Comment:
`python_removed_class` doesn't exist. Those are doxy ALIAS that translate to html: https://github.com/xbmc/xbmc/blob/388ff63e2659186444d195c236369d17517964cd/docs/doxygen/Doxyfile.doxy#L252

Reply:
would it be as simple as adding a new `python_removed_class` line there?
i'm hardly familiar with doxygen code...

Reply:
Probably add the 3 functions instead of the class, result is pretty much the same. And python classes are not deleted everyday :)

Reply:
ok, updated